extensions/hibernate-orm/deployment/src/main/java/io/quarkus/hibernate/orm/deployment/HibernateOrmProcessor.java

```diff
@@ -664,6 +664,20 @@ public void registerStaticMetamodelClassesForReflection(CombinedIndexBuildItem i
         }
     }
 
+    /*
+     * Enable reflection for methods annotated with @InjectService,
+     * such as org.hibernate.engine.jdbc.cursor.internal.StandardRefCursorSupport.injectJdbcServices.
+     * We only deal with built-in service classes here;
+     * integrators relying on @InjectService must declare reflection on their own classes.
```

Comment:
Is there a reason why you don't scan the index for additional ones? Wouldn't cost much.

Reply:
It seems Hibernate ORM is not indexed by Jandex for some reason...

Reply:
Ah, you mean for the integrators. Sure I can do it. I don't particularly want to test this, though :)

Reply:
Done

Reply:
Yes, I mean for the integrators. And sure, no need to test it. I would add everything to a `Set` to be sure we don't have duplicates in case someone indexes ORM for some reason.

Reply:
> I would add everything to a Set to be sure we don't have duplicates in case someone indexes ORM for some reason.

Done